config/locales/en.yml

```diff
@@ -124,11 +131,6 @@ en:
           goals_against: "points against"
           goals_difference: "points difference"
           points: "league points"
-    join_team_event: "Join event %{event} as a team"
-    max_teams:
-      infinite: "unlimited"
-    overview:
-      not_available_for_leagues: 'No overview available for leagues.'
```

Comment:
how come the number of changed lines in both translation files is not the same?

Reply:
This diff looks very strange to me too. It's somhow not showing the lines that have been moved but deleting the lines in between and inserting them before.

Reply:
Why were there even changes made to translation files in this PR?

Reply:
Because I moved the ranking view to the leagues folder.